src/pages/TeamSettings/CloudHooks.vue

```diff
@@ -250,6 +250,16 @@ export default {
         this.hooksLoaded = true
       },
       update(data) {
+        // This forces type conversion for email lists from the API
+        data.cloud_hook
+          .filter(c => c.type == 'EMAIL')
+          .forEach(
+            c =>
+              (c.config['to'] =
+                typeof c.config['to'] === 'string'
+                  ? c.config['to']
+                  : c.config['to'].join(','))
+          )
```

Comment:
Can you explain why this is needed?

Reply:
It shouldn't be needed anymore since we're always assuming strings in Cloud - haven't had a chance to remove yet

Reply:
👍